### ISSUE TITLE
Modify test part of CI build to work in parallel

### ISSUE THREAD

**Issue by LADSoft** [LADSoft/OrangeC#129]
As part of #85 it was determined that the 'testing' portion of the Appveyor CI build does not currently work well in parallel builds.   Rather than have that issue go on indefinitely, this issue is being opened to track that.

**Comment by GitMensch** [comment]
Just a question: Do you consider this working? If yes: I suggest to use the multiple jobs option throughout all tests. If not: just postpone until the other tools are fine.

**Comment by LADSoft** [comment]
I haven't gotten it working yet.   I've seen it not work on my computer but can't duplicate at the moment...  I'm gonna let it go for the moment.

**Comment by LADSoft** [comment]
I bumped it so that it does the multiple jobs for the tests now.
